### DISCUSSION TITLE
CDP MODE - iframes

### DISCUSSION THREAD

**Discussion by Eduardo-Qinttus** [seleniumbase/SeleniumBase#3292] [General]
is it possible to handle iframes in cdp mode?

**Comment by mdmintz** [comment]
Use `sb.cdp.nested_click(parent_selector, selector)`

https://github.com/seleniumbase/SeleniumBase/blob/d8d02cd4740e700ef6ea3b133bbdc9830e728cf4/examples/cdp_mode/ReadMe.md?plain=1#L335

(Where the `parent_selector` is that of the `iframe`)

**Reply by Eduardo-Qinttus** [reply]
Thank You

**Reply by ekramia1973** [reply]
What to set for parent_selector?

**Reply by Eduardo-Qinttus** [reply]
and if the element is below two iframes? how can i click on it?
